### DISCUSSION TITLE
Office 365 - Could not connect to SMTP host - RANDOMLY

### DISCUSSION THREAD

**Discussion by bbtrash** [PHPMailer/PHPMailer#2536] [Q&A] [answered]
Hello, 
we are sometimes experiencing "could not connect to SMTP host" error. Everything is ok, but sometimes PHPMailer ends up with this error. Currently I am blaming MS servers. I am curious if somebody have same issue or if there is any workaround to solve this occasional behavior.

**Unsuccessful try to send email:**

SERVER -> CLIENT: 220 VI1PR0102CA0008.outlook.office365.com Microsoft ESMTP MAIL Service ready at Mon, 25 Oct 2021 08:57:28 +0000
CLIENT -> SERVER: EHLO our.web
SERVER -> CLIENT: 250-VI1PR0102CA0008.outlook.office365.com Hello [our.ip]250-SIZE 157286400250-PIPELINING250-DSN250-ENHANCEDSTATUSCODES250-STARTTLS250-8BITMIME250-BINARYMIME250-CHUNKING250 SMTPUTF8
CLIENT -> SERVER: STARTTLS
SERVER -> CLIENT: 220 2.0.0 SMTP server ready
SMTP Error: Could not connect to SMTP host.
CLIENT -> SERVER: QUIT
SERVER -> CLIENT:
SMTP ERROR: QUIT command failed:
SMTP connect() failed. https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting

**After two minutes next try was successful:**

SERVER -> CLIENT: 220 VI1PR09CA0049.outlook.office365.com Microsoft ESMTP MAIL Service ready at Mon, 25 Oct 2021 08:58:08 +0000
CLIENT -> SERVER: EHLO our.web
SERVER -> CLIENT: 250-VI1PR09CA0049.outlook.office365.com Hello [our.ip]250-SIZE 157286400250-PIPELINING250-DSN250-ENHANCEDSTATUSCODES250-STARTTLS250-8BITMIME250-BINARYMIME250-CHUNKING250 SMTPUTF8
CLIENT -> SERVER: STARTTLS
SERVER -> CLIENT: 220 2.0.0 SMTP server ready
CLIENT -> SERVER: EHLO our.web
SERVER -> CLIENT: 250-VI1PR09CA0049.outlook.office365.com Hello [our.ip]250-SIZE 157286400250-PIPELINING250-DSN250-ENHANCEDSTATUSCODES250-AUTH LOGIN XOAUTH2250-8BITMIME250-BINARYMIME250-CHUNKING250 SMTPUTF8
CLIENT -> SERVER: AUTH LOGIN
SERVER -> CLIENT: 334 VXNlcmhbWU6
CLIENT -> SERVER: dHZAdp2LmN6
SERVER -> CLIENT: 334 UGFzc3dvcQ6
CLIENT -> SERVER: TWljYSw1ODIyNm1MWw=
SERVER -> CLIENT: 235 2.7.0 Authentication successful
.... and so on - boring stuff because send was ok ...  

Thanks a lot

**Comment by Synchro** [comment, marked as answer]
This is the same question you posted as an issue, and the same answer applies.

**Reply by bbtrash** [reply]
I did not posted anything to issues, because this is not PHPMailer issue I guess. May you provide link to issues where is answer please ?

**Reply by bbtrash** [reply]
https://github.com/PHPMailer/PHPMailer/issues/2534
